spec/lib/jobs_spec.rb

```diff
@@ -88,6 +88,13 @@
       end
     end
 
+    context 'when the search phrase includes "federal" or "gobierno" ' do
```

Comment:
@peggles2 , I think this spec context should be more generic:
`context 'when the search phrase does not include job-related keyword' do`

That makes it more explicit that the logic applies to any non-job-related keyword, not just those two.